### ISSUE TITLE
Try to improve the searching feature.

### ISSUE THREAD

**Issue by 2tanayk** [gauravraj0510/PopChat#3]
I understand there is no regex in firebase.So basically what you can try is maybe whatever the user searches get it in a String and then use Java Regex and compare it with what user has searched  against all the contacts and appropriately return results,best of luck :).

**Comment by gauravraj0510** [comment]
Yeah thats a brilliant suggestion i will try doing that. Thanks a lot!
